packages/ktracker/PatternDBGen.cxx

```diff
@@ -0,0 +1,375 @@
+/**
+ * \class PatternDBGen
+ * \brief General purposed evaluation module
+ * \author Haiwang Yu, yuhw@nmsu.edu
+ *
+ * Created: 08-27-2018
+ */
+
+
+#include "PatternDBGen.h"
+
+#include <interface_main/SQHit.h>
+#include <interface_main/SQHit_v1.h>
+#include <interface_main/SQMCHit_v1.h>
+#include <interface_main/SQHitMap_v1.h>
+#include <interface_main/SQHitVector_v1.h>
+#include <interface_main/SQEvent_v1.h>
+#include <interface_main/SQRun_v1.h>
+#include <interface_main/SQSpill_v1.h>
+#include <interface_main/SQSpillMap_v1.h>
+
+#include <ktracker/SRecEvent.h>
```

Comment:
This line should be changed to #include "SRecEvent.h".  Otherwise a build from scratch fails.

Reply:
Thanks! I will change it in the next PR.